packages/suite/src/actions/suite/storageActions.ts

```diff
@@ -20,6 +20,8 @@ import type { PreloadStoreAction } from 'src/support/suite/preloadStore';
 import { GraphData } from 'src/types/wallet/graph';
 import { deviceGraphDataFilterFn } from 'src/utils/wallet/graph';
 import { selectCoinjoinAccountByKey } from 'src/reducers/wallet/coinjoinReducer';
+import { selectCoinsLegacy } from '@suite-common/wallet-core';
+import { useSelector } from 'src/hooks/suite';
```

Comment:
I don't think we can use redux useSelector hook outside components in action creator, can we?

Reply:
nope <img width="695" alt="image" src="https://github.com/trezor/trezor-suite/assets/3729633/e558ac4e-76a2-4a49-b703-a6501ecdf4e5">

Reply:
f0796072864f0ae73f74e1d235b43c4dc83a2dfb

Reply:
it could be a fixup if it fixes a specific commit...